application/app_config.py

```diff
@@ -4,7 +4,7 @@
 
 
 class AppConfig(object):
-    PROJECT_DIR = os.path.realpath(os.path.join(os.path.dirname(__file__), "..", "project_dir"))
+    PROJECT_DIR = os.path.realpath(os.path.join(os.path.expanduser('~'),"Documents" , "TrafficAnalysis", "project_dir"))
```

Comment:
This is a huge change to how things work on my system.  Currently, my project_dir is stored at 

{path_to_gui_folder}/SantosGUI/project_gui

This change makes it ~/Documents/TrafficAnalysis/project_dir, which is not a folder I have on my Mac setup